res/enkf/export/gen_kw_collector.py

```diff
@@ -21,14 +21,20 @@ def getAllGenKwKeys(ert):
         return ert.getKeyManager().genKwKeys()
 
     @staticmethod
-    def loadAllGenKwData(ert: EnKFMain, case_name, keys=None, realization_index=None):
+    def loadAllGenKwData(
+        ert: EnKFMain,
+        case_name,
+        keys=None,
+        realization_index=None,
+        read_only: bool = False,
+    ):
         """
         @type ert: EnKFMain
         @type case_name: str
```

Comment:
`@rtype` is deprecated in favor of type hints, and the docstring needs to be updated generally.